apm-agent-core/src/test/resources/logback-test.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[question] why do we have to use `logback-test.xml` here an not `logback.xml` ?

Reply:
We don't have to, but it is very specific for tests (the schema validation library) and it works...